src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
 
-    public int total = -1;
+    public static final int ERROR = -1;
+    public static final int MAX_SIZE = 12;
+    private final int[] numbers = new int[MAX_SIZE];
+
+    public int total = ERROR;
+
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == ERROR;
     }
 
     public boolean isFull() {
-        return total == 11;
-    }
+        return total == MAX_SIZE-1;
```

Comment:
To co wyżej, jeśli dodasz stałą INITIAL_TOTAL_VALUE będziesz mógł napisać total == MAX_SIZE + INITIAL_TOTAL_VALUE